desktop_version/src/UtilityClass.h

```diff
@@ -68,6 +68,25 @@ void _VVV_between(
         sizeof(middle) \
     )
 
+#ifndef __has_attribute
+#   define __has_attribute(x) 0
+#endif
+
+#if __has_attribute(__fallthrough__)
```

Comment:
This might need to be named something less generic like `VVV_case_fallthrough` so that it doesn't accidentally step on a commonly-used word.

Reply:
Done.